### ISSUE TITLE
DO NOT USE THE BOT THESE DAYS! HIGHLY POSSIBLE TO BE BANNED!!!!!!!!!!

### ISSUE THREAD

**Issue by xulusjb** [xulusjb/PUBG#16]

**Comment by AlbCM** [comment]
![image](https://user-images.githubusercontent.com/10161576/38132798-3db3cdce-33d2-11e8-9d01-6f224c222b39.png)

**Comment by AlbCM** [comment]
How is it possible :( ?

**Comment by ctrlsam** [comment]
ikr, how the fuck is this possible and why?!?!?!

**Comment by ctrlsam** [comment]
For those that have been banned, it seems many others (that aren't hacking) are being banned too, all within the last 10 hours.
Examples:
- https://steamcommunity.com/app/578080/discussions/1/1697169163399711488/
- https://fragbite.com/pubg/news/42261/professional-pubg-player-jeemzz-banned--updated
- https://steamcommunity.com/app/578080/discussions/1/1697169163399472164/
- https://steamcommunity.com/app/578080/discussions/1/1697169163398932733/
- http://steamcommunity.com/app/578080/discussions/1/1697169163400255784/
- https://steamcommunity.com/app/578080/discussions/1/1698294337772770745/?ctp=1

Response to a pro player getting banned: "There is a big misunderstanding here. The account was not banned because of player reports. It was **falsely banned by one of our new measures which we will fix soon.**"

PUBG Added on a Tweet: "If you feel you were incorrectly banned please submit an appeal on our support website -https://support.playbattlegrounds.com/ "

If we all send appeals, hopefully they will realize that they banned the innocent.

**Comment by hiroto-takatoshi** [comment]
The detection algorithm mainly targets the advertising bot accounts - whose ID contains contact to the aimbot sellers, broadcasting audio CMs in the public channel in the game lobby and quit on plane. 

The feature targeted might be staying in one game for too short or being online for too long.

To distinguish from them, try showing some proof of recent game activities (screenshots, screencasts or online database logs).

**Comment by hiroto-takatoshi** [comment]
To follow, I found out that the banned accounts have a same pattern: they quited on the plane for too many matches. This is why the other account I used for bp farming was not banned.

**Comment by caobidawang** [comment]
现在能用吗，今天凌晨挂了一早上也没出问题。
所以说只是因为官方算法是检测，广告ID以及长时间上飞机秒退，发广告。才会被封禁吗？

**Comment by caobidawang** [comment]
已经被BAN，暂时别用了

**Comment by ctrlsam** [comment]
**Response from PUBG Support**
_It seems very automated and looks like no real investigation really took place :(_

Hello Survivor,

Thank you for writing to PUBG Support.

It looks like you are seeing a restriction error message on the account.

We understand that this may be a very frustrating experience.However, please understand that a restriction is placed because we have determined that there was a violation of our terms and services.

Please note that even without having any programs specifically for “hacking/cheating” there are other programs or actions that can violate our Rules of Conduct and cause a permanent restriction. 

- Software that bypasses the services in the game
- Programs that alter the game in any way (even if it is not for cheating)
- Cheats, macros, etc. for other games and not for PUBG is still a violation of the game’s services

Having purchases, low KPD, short gameplay time, etc. does not exclude any accounts from restrictions. 

We can assure you that the account was blocked because there was a violation discovered on the account. We apologize but we cannot provide any specifics on the details of the violation.

We have also implemented a 24 hour investigation period to ensure that there are no false bans. Therefore, we do not accept any requests for rechecking account restrictions. However, we also constantly check our systems for false bans and if found, we correct any issues internally.

We ask for your understanding of this situation.We will continue our work to make sure that no erroneous restrictions are made and continue creating a fair gaming environment for all BATTLEGROUNDS players.

Please let us know if you have any other feedback, questions or comments. I'll be happy to assist you in any way I can.

**Comment by hiroto-takatoshi** [comment]
... and if you reply to this autoreply it will auto warn you possibilities of a forever ban if you keep replying

**Comment by ctrlsam** [comment]
yup. Its not like 100 year ban and perma ban makes much difference anyway

**Comment by Falubii** [comment]
Well, as i understood you will not be unbanned? Or they are doing some research to prove you innocent?

**Comment by Falubii** [comment]
![image](https://user-images.githubusercontent.com/38011287/38204088-1f5f3680-36aa-11e8-9b88-caa6df95164e.png)
I got only this from battleye.com(

**Comment by ctrlsam** [comment]
I followed up with an email stating i wanted them to do a better investigation but then they responded saying pretty much the same stuff, overall expressing their anticheat is flawless. I will never be unbanned and forever have a ban on my clean steam account ( http://steamcommunity.com/id/sillysam )

**Comment by ctrlsam** [comment]
@Falubii I see you have contacted battle eye directly, please let us know how that goes as I contacted PUBG support.

**Comment by Falubii** [comment]
![image](https://user-images.githubusercontent.com/38011287/38263781-a881b710-3779-11e8-83c3-dbd9a2cdf491.png)
Well it's what i got from pubg support. Now need wait 100 years or to buy a new copy

**Comment by ctrlsam** [comment]
I have been unbanned 👍

**Comment by Falubii** [comment]
Lol, how did you do that??!?!?!

**Comment by Falubii** [comment]
![image](https://user-images.githubusercontent.com/38011287/38421725-cac7bf2a-39b0-11e8-89b5-84ace15a5ced.png)
I was unbanned too, sry for Russian, it was said that i was unbanned
![image](https://user-images.githubusercontent.com/38011287/38421755-e94dca20-39b0-11e8-8bf3-b1b51c2a910b.png)

**Comment by ctrlsam** [comment]
It's actually quite funny. I was telling my friend about the ban after I showed you guys my chat with the support. My friend supported me and decided to express his feelings to PUBG support too. He talked about how "Sam (me) got a lucky shot just before he got banned" (even though there is no lucky shot) and then bam!

**Comment by ctrlsam** [comment]
@Falubii Ayyy!

**Comment by ctrlsam** [comment]
@Falubii wanna play :D

**Comment by TahaKinik** [comment]
Do you guys have any idea why are we unbanned ?

**Comment by Falubii** [comment]
@didonik You too? well i think it was mistake, they could upgrade anti cheat and he failed and banned everyone for our bot actions

**Comment by wavyjetski** [comment]
So is it safe to use the bot now as you got unbanned?

**Comment by TahaKinik** [comment]
@Falubii I don't really think it was a mistake. @wavyjetski  Idk.

**Comment by ctrlsam** [comment]
@wavyjetski I would not recommend at this time. You should make mouse movements smooth as well as doing some random stuff when on the ground.

**Comment by Falubii** [comment]
@didonik So if it's not mistake,we are piece of lucky shit? That all of us were unbanned in the same day? Well everything possible, but Imao it was some problem in anticheat and it reacted on our not obvious movement and exit from match during some period of time.

**Comment by Falubii** [comment]
![image](https://user-images.githubusercontent.com/38011287/38455455-05d53bbc-3a81-11e8-937f-e8b9bd7dd0d1.png)

**Comment by TahaKinik** [comment]
That's interesting. Maybe they wanted to warn us like " We now you're botting it's our way to warn you." and they might update their terms and services(I don't know how does that work)

**Comment by hiroto-takatoshi** [comment]
https://www.reddit.com/r/PUBATTLEGROUNDS/comments/8bet6k/80bots_in_eu_fpp_games_jumping_and_proneing/
They will be targeting this.

**Comment by TahaKinik** [comment]
80 bot in 1 game is just too much. Probably because of that they will target all the afk bots.

**Comment by viskyy** [comment]
Just a question. When everyone was using the 0328 version, did you guys use a lot of the derank option or just mainly farming? Because I used a lot of the derank before I got banned.

**Comment by TahaKinik** [comment]
I was using for farming.

**Comment by hiroto-takatoshi** [comment]
We should have suggested you to jump off the plane, and quit until landed. Didn't know the ban would come though.
